presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoInstaller.java

```diff
@@ -85,21 +93,56 @@ private static void installUsingRpm(String pathToRpm, boolean checkDependencies)
         if (installRpm != 0) {
             throw new PrestoManagerException("Failed to install Presto", installRpm);
         }
-        addTpchCatalog();
+        addConfigFiles(CONFIG_DIR, DEFAULT_CONFIG_DIR);
+        addConnectors(CATALOG_DIR, DEFAULT_CATALOG_DIR);
```

Comment:
These could be put in their own function (`addDefaultFiles`?), but this is fine, too.